src/main/java/net/snowflake/client/core/SFArrowResultSet.java

```diff
@@ -592,21 +592,22 @@ private Object getObjectRepresentation(int columnIndex, boolean withString) thro
     String jsonString = withString ? converter.toString(index) : null;
     boolean isStructuredType = resultSetMetaData.isStructuredTypeColumn(columnIndex);
     if (isVarcharConvertedStruct(type, isStructuredType, converter)) {
-        return new StructObjectWrapper(jsonString, createJsonSqlInput(columnIndex, obj));
+      return new StructObjectWrapper(jsonString, createJsonSqlInput(columnIndex, obj));
```

Comment:
is it good PR name for the provided change?

Reply:
Fixed

Reply:
Let's add also jira id

Reply:
ok